### PR TITLE
EP-261/Android: Video Playback Started

### PR DESCRIPTION
**Pull request by hadia**
# 📲 What
add Video Playback Started segment event

# 🤔 Why

A user clicks the play button on the main video on a project page

# 🛠 How

Add video Properties
Screen Shot 2021-03-18 at 5 27 26 PM

Add Event Handler in Video ViewModel
image

Update project property "Launched_at" and Deadline to time stamp to solve violations 
image

Add test
image

# 👀 See
image

# 📋 QA

open Project and click the video Play button
# Story 📖

https://kickstarter.atlassian.net/browse/EP-261

### CODE REVIEW

File: app/src/main/java/com/kickstarter/viewmodels/VideoViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package com.kickstarter.viewmodels
```
Comment (Arkariang) [inline app/src/main/java/com/kickstarter/viewmodels/VideoViewModel.kt:1]: 🥳  more migrations to Kotlin! 👏

File: app/src/test/java/com/kickstarter/libs/LakeTest.kt [hunk 2]
```diff
@@ -475,6 +475,31 @@ class LakeTest : KSRobolectricTestCase() {
         this.lakeTest.assertValues("Project Page Pledge Button Clicked")
     }
 
+    @Test
+    fun testVideoProperties() {
```
Comment (Arkariang) [inline app/src/test/java/com/kickstarter/libs/LakeTest.kt:479]: This is great @hadia is perfect! just a small thing, can you move this test from `LakeTest` to `SegmentTest` or duplicate it as you wish, Lake client will be deleted some time in the near future alongside with the tests for it

**Review by Arkariang — APPROVED** [review]
🏆  looks great!

File: app/src/test/java/com/kickstarter/libs/LakeTest.kt (hunk 2 above)
Comment (hadia) [reply app/src/test/java/com/kickstarter/libs/LakeTest.kt:479]: sure

**Pull request merged**